### ISSUE TITLE
Rule Proposal: single object in class array

### ISSUE THREAD

**Issue by privatenumber** [vuejs/eslint-plugin-vue#1093]
**Please describe what the rule should do:**

The rule should enforce that only one object is used in an array of classes

**What category should the rule belong to?**

- [x] Enforces code style
- [ ] Warns about a potential error
- [ ] Suggests an alternate way of doing something
- [ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**
```vue
<template>
	<div :class="[{ a: true }, { b: true }, 'c']" />
</template>
```

Can be fixed to:
```vue
<template>
	<div :class="[{ a: true, b: true }, 'c']" />
</template>
```

Can open a PR if proposal is approved.

**Comment by ota-meshi** [comment]
Thank you for proposing the rule.

I like that rule you proposed.
